### PR TITLE
Update Korean localization

### PR DESCRIPTION
**Pull request by Canorus**
- Fix typo
- Change 'redo' translation to what majority agreed as the most natural one
- Add notification preference translation
- Add time format translation

### CODE REVIEW

File: src/config/locales/ko/translation.json [hunk 1]
```diff
@@ -100,13 +105,13 @@
...
       "notify": {
-        "title": "Notify",
-        "reply": "Reply:",
-        "reblog": "Reblog:",
-        "favourite": "Favourite:",
-        "follow": "Follow:"
+        "title": "푸쉬 알림",
+        "reply": "답장:",
+        "reblog": "부스트:",
+		"favourite": "즐겨찾기:",
```
Comment (h3poteto) [inline src/config/locales/ko/translation.json]: What is this space?

**Comment by h3poteto** [comment]
Yes, I hope that each language specialists will translate it.

**Comment by Canorus** [comment]
well wish there's some way to send notification, desirably automatically when translation file is changed, to whoever contributed in translation. I didn't even know there was new items to translate…

**Comment by Canorus** [comment]
I didn't have to close and pull a new request thanks to github, and github gracefully resolved the conversation as well.

**Comment by h3poteto** [comment]
No problem, thank you for your translation and fix.

**Pull request merged**